### PR TITLE
TEL-205-create-tab-for-analysis-with-graphs-and-stats-subtab

### PR DESCRIPTION
**Pull request by kr1s-r**

### CODE REVIEW

File: packages/client/src/components/tabs/AnalysisTab.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  {
+    id: 0,
+    name: "Graphs",
+  },
+
+  {
+    id: 1,
+    name: "Stats",
+  },
+];
```
Comment (burtonjong) [inline packages/client/src/components/tabs/AnalysisTab.tsx:28]: can we move these to TabRoutes.tsx instead and import them from there

File: packages/client/src/components/tabs/AnalysisTab.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
+import Image from "next/image";
+import React, { useState } from "react";
+
+import { ThemeProvider } from "@emotion/react";
+import { Tab, Tabs, createTheme } from "@mui/material";
+
+type TabProps = {
+  id: number;
+  name: string;
+};
```
Comment (burtonjong) [inline packages/client/src/components/tabs/AnalysisTab.tsx:10]: Move this to TabRoutes.tsx again

**Review by burtonjong — CHANGES_REQUESTED** [review]

**Review by burtonjong — DISMISSED** [review]
LGTM

File: packages/client/src/components/tabs/AnalysisTab.tsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
+import Image from "next/image";
+import React, { useState } from "react";
+
+import { tabs } from "@/objects/TabRoutes";
+import { ThemeProvider } from "@emotion/react";
+import { Tab, Tabs, createTheme } from "@mui/material";
+
+type TabContentProps = {
+  children: React.ReactNode;
```
Comment (brian-ngyn) [inline packages/client/src/components/tabs/AnalysisTab.tsx:9]: can you use PropsWithChildren

**Review by burtonjong — APPROVED** [review]
LGTM

**Review by alexwhelan12 — APPROVED** [review]
LGTM!

**Pull request merged**